docs/reference/requirement-sets/open-browser-window-api-requirement-sets.md

```diff
@@ -0,0 +1,41 @@
+ms.date: 09/16/2020
+ms.prod: non-product-specific
+localization_priority: Normal
+---
+
+# Open Browser Window API requirement sets
+
+Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office host supports APIs that an add-in needs. For more information, see [Office versions and requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets).
+
+The OpenBrowserWindow API set enables add-ins to open a browser to accomplish tasks that cannot always be done in the sandboxed webview control within the add-in itself; for example, downloading a PDF file when the webview control is provided by Microsoft Edge.
+
+Office Add-ins run across multiple versions of Office. The following table lists the OpenBrowserWindow API requirement sets, the Office host applications that support that requirement set, and the build or version numbers for the Office application.
+
+|  Requirement set  | Office 2013 on Windows or later<br>(one-time purchase) | Office on Windows<br>(connected to Office 365 subscription) |  Office on iPad<br>(connected to Office 365 subscription)  |  Office on Mac<br>(connected to Office 365 subscription)  | Office on the web  |  Office Online Server  |
+|:-----|-----|:-----|:-----|:-----|:-----|:-----|:-----|
+| OpenBrowserWindowApi 1.1  | N/A | Version 1810 (Build 16.0.11001.20074) or later | 16.0.0.0 or later | 16.0.0.0 or later | N/A | N/A|
+
+To find out more about versions, build numbers, and Office Online Server, see:
+
+- [Version and build numbers of update channel releases for Office 365 clients](https://support.office.com/article/version-and-build-numbers-of-update-channel-releases-ae942449-1fca-4484-898b-a933ea23def7)
+- [What version of Office am I using?](https://support.office.com/article/What-version-of-Office-am-I-using-932788b8-a3ce-44bf-bb09-e334518b8b19)
+- [Where you can find the version and build number for an Office 365 client application](https://support.office.com/article/version-and-build-numbers-of-update-channel-releases-ae942449-1fca-4484-898b-a933ea23def7)
+- [Office Online Server overview](/officeonlineserver/office-online-server-overview)
+
+## Office Common API requirement sets
+
+For information about Common API requirement sets, see [Office Common API requirement sets](office-add-in-requirement-sets.md).
+
+## OpenBrowserWindowAPI 1.1
+
+The OpenBrowserWindowAPI 1.1 is the first version of the API. For details about the API, see the [Office.context.ui
+](/javascript/api/office/office.context.ui) reference topic.
+
+## See also
+
+- [Office versions and requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets)
+- [Specify Office hosts and API requirements](/office/dev/add-ins/develop/specify-office-hosts-and-api-requirements)
+- [Office Add-ins XML manifest](/office/dev/add-ins/develop/add-in-manifests)
```

Comment:
Since these are in the same repo, should use relative path e.g. ../develop/add-in-manifests.md (not sure how many levels you need to go up). Here and other places in this file.